fcd/cpu/x86_regs.h

```diff
@@ -23,7 +23,7 @@
 #define x86_regs_h
 
 #include <cstdint>
-#include <x86.h> // capstone/x86.h
+#include <capstone/x86.h>
```

Comment:
This is incompatible with the current source release of Capstone. That will be fixed whenever the `next` branch is merged into the `master` branch (and released).